prefect_aws/ecs.py

```diff
@@ -531,12 +543,63 @@ async def run(self, task_status: Optional[TaskStatus] = None) -> ECSTaskResult:
         )
 
         return ECSTaskResult(
-            identifier=task_arn,
+            identifier=identifier,
             # If the container does not start the exit code can be null but we must
             # still report a status code. We use a -1 to indicate a special code.
             status_code=status_code or -1,
         )
 
+    @sync_compatible
+    async def kill(self, identifier: str, grace_period: int = 30) -> None:
+        """
+        Kill a task running on ECS.
+
+        Args:
+            identifier: A cluster and task arn combination. This should match a value
+                yielded by `ECSTask.run`.
+        """
+        if grace_period != 30:
+            self.logger.warning(
+                "Kill grace period of {grace_period}s requested, but AWS does not "
+                "support dynamic grace period configuration. "
+                "See https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs-agent-config.html for configuration of grace periods."  # noqa
+            )
```

Comment:
```suggestion
            self.logger.warning(
                "Kill grace period of %ss requested, but AWS does not "
                "support dynamic grace period configuration. "
                "See https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs-agent-config.html for configuration of grace periods.",  # noqa
                grace_period
            )
```

Reply:
We do not use that (ik, more correct) style anywhere in Prefect and notably nowhere in this file. I think we should stick to the style of the current file, at least. We can change these to be the proper "delayed" formats later. 

This did make me notice that there's not a test for this though!

Reply:
Okay; that's fine with me. In that case, I think the string needs to be converted to f-string if I'm not mistaken.

Reply:
👍 yeah good catch.